### PR TITLE
Features/86 update debian version

### PR DESCRIPTION
**Pull request by santi-eidu**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions
* Remember to set **idealista:develop** as base branch;
* Add "idealista/benders" as reviewer

### Description of the Change

Change travis-ci to use focal instead of xenial. Change jobs to use Debian bulleye instead of stretch.
Update ansible to v5.2.0 and ansible-lint to v5.3.2
Change molecule in template to use Debian bulleye instead of stretch.

### Benefits

Add support for Debian Bulleye in Molecule.
Solve issue with ansible-lint v5.1.2.
Fix passing build travis image in README.md.

### Possible Drawbacks

No drawbacks

### Applicable Issues

#86

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a changelog](https://github.com/olivierlacan/keep-a-changelog).
 
 ## Unreleased
+## [2.9.0](https://github.com/idealista/cookiecutter-ansible-role/tree/2.9.0)
+### [Full Changelog](https://github.com/idealista/cookiecutter-ansible-role/compare/2.8.1...2.9.0)
+### Changed
+- *[#86](https://github.com/idealista/cookiecutter-ansible-role/issues/86) Upgrade molecule config to test under debian bullseye* @sperez
```
Comment (blalop) [inline CHANGELOG.md:9]: Remember to use your github user :)

Comment (santi-eidu) [reply CHANGELOG.md]: Fixed with commit https://github.com/idealista/cookiecutter-ansible-role/pull/90/commits/fdbc7c3908f3013606fad0b1af3a41524eb2e1a7

**Review by blalop — APPROVED** [review]

File: {{cookiecutter.app_name}}_role/molecule/default/Dockerfile.j2 [hunk 2]
```diff
@@ -8,5 +8,5 @@ FROM {{ item.image }}
 
 # install minimal packages for debian slim images
 RUN apt-get update && \
-    apt-get install -y python sudo bash ca-certificates iproute2 systemd systemd-sysv python-pip && \
+    apt-get install -y python sudo bash ca-certificates iproute2 systemd systemd-sysv python3-pip && \
```
Comment (blalop) [inline {{cookiecutter.app_name}}_role/molecule/default/Dockerfile.j2:11]: why not `python3`?

**Review by blalop — APPROVED** [review]

**Review by emepege — APPROVED** [review]
Way to go! :)

File: {{cookiecutter.app_name}}_role/molecule/default/Dockerfile.j2 (hunk 2 above)
Comment (santi-eidu) [reply {{cookiecutter.app_name}}_role/molecule/default/Dockerfile.j2]: I did not check that, remove python2 and add python3 to Dockerfile.

**Review by ultraheroe — APPROVED** [review]

**Review by jperera — APPROVED** [review]

**Pull request merged**